### DISCUSSION TITLE
Where & how to contact to find out instructions around use of GitHub Actions & its logo

### DISCUSSION THREAD

**Discussion by KaykoSRS** [community/community#33272] [New to GitHub]
Hi, could anyone here advise where and how to contact to get instructions around the use of and logo of GitHub Actions?

I need to confirm the guidelines or instructions to use GitHub Actions and its logo in the contents for a marketing campaign.  I believe there will be need for me to talk to a proper team in charge at GitHub of this area.

Thought of contacting GitHub but could not find where and how to contact (I am not entitled to create and submit a ticket to send in inquiries).

**Comment by queenofcorgis** [comment]
Hi there! 

Are you not able to create a ticket with [Support](support.github.com)? That's who you would contact providing specific details on what you want to use, how you will use it, and for how long. Your request will be reviewed and determined based on the details provided.

Cheers,
Lili

**Reply by KaykoSRS** [reply]
Hi Lili, thank you for your response.
Sorry to say, I am not able to submit a Support ticket and posted a question here.

Could you or anyone help me or guide me how to contact Support without raising a support ticket?

**Reply by queenofcorgis** [reply]
Hi there - can you elaborate on why you are unable to submit a ticket so I can better assist? Thanks!

**Reply by KaykoSRS** [reply]
Hi Lili, below is the path I tried and the result I am not able to create a support ticket to submit.
Sign in and reach to the page
> GitHub Support
> My Tickets
> New Ticket
> "How can we help?" page https://support.github.com/request
> Select a topic "Copy right, trade mark, ... "
> https://docs.github.com/en/site-policy/content-removal-policies/submitting-content-removal-requests
> No answer I can find here and select "Contact Support"
> Back to GitHub Support and endless loop, I can find no way to create and submit a ticket.

**Reply by byrneh** [reply]
@KaykoSRS the [Contact us] link takes you to [https://support.github.com/request](https://support.github.com/request)
Choose a topic to help us route your request quickly. Dd you try selecting **Other Topic** and enter details there.

[https://github.com/logos](https://github.com/logos) provides some guidelines for those times when you want to use logos to show off your GitHub integration or link back to GitHub. GitHub logos represent only _GitHub and should not be used to represent you or your projects, products, or company_.

**Reply by KaykoSRS** [reply]
Hi @byrneh, thank you for your note.

I tried clicking [Contact us] to https://support.github.com/request and tried selecting other topics.  Some ended up with endless loop.  Some topics, for example, 'Account or billing issues' moved to a submission form page, but the scope of the topic is too different.

If it is OK to create and submit from there, I will do so.  Could  you please advise?

And I have already checked https://github.com/logos and guidelines. Given that said, because of below, I feel I need to get connected to support or marketing;
- Use of GitHub Actions logo and mention in marketing leads-gen action.
- Conversations in this https://github.com/orgs/community/discussions/22177

**Reply by KaykoSRS** [reply]
Hi @byrneh or anyone here, sorry for keep on bothering you but I need your support to find out the way to confirm, and could you please advise how I should take next action?
